extra/service/legion-linux.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Apply Fan Curve to Lenovo Legion Laptop depending of Power State and Charger Mode (LenovoLegionLinux kernel Module)
+DefaultDependencies=no
+StopWhenUnneeded=yes
+After=systemd-logind.service
+
```

Comment:
add `StartLimitIntervalSec=0` here to prevent limit-hit

Reply:
Yes that a good ideia